### PR TITLE
feat: add component create hero banner

### PR DESCRIPTION
**Pull request by adzharamrullah**
## Changes

- feat: add component create hero banner
- feat: add drop file upload
- feat: add component base modal
- feat: add component base toast
- feat: add funtionality seach datatable
- fix: remove header description on datatable

## Evidence
title: feat: add component create hero banner
project: Desa Digital
participants: @adzharamrullah @Ibwedagama @maulanayuseph @yoslie

### CODE REVIEW

File: components/AddHeroBanner/index.vue [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+          File yang didukung adalah .svg, .jpg dan .png
+        </div>
+      </div>
+      <div
+        :class="{
+          'form-add-hero-banner__box-upload-file': true,
+          'form-add-hero-banner__box-upload-file--dragover': isDragOver
+        }"
+        @drop.prevent="onFileDrop"
+        @drag.prevent
```
Comment (Ibwedagama) [inline components/AddHeroBanner/index.vue:28]: hi @adzharamrullah is this event is not bonded to any action on purpose?

File: components/AddHeroBanner/index.vue [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+          this.dropFileErrorMessage = 'Gambar anda melebihi ukuran maksimal'
+          this.isDragOver = false
+        } else {
+          this.isDragOver = true
+          this.dropFileErrorMessage = ''
+          this.setFile(e.dataTransfer.files[0])
+        }
+      } else {
+        this.form.fileImage = null
+        this.dropFileErrorMessage = 'Maaf file yang anda memasukan tidak didukung'
```
Comment (Ibwedagama) [inline components/AddHeroBanner/index.vue]: I think this error message need some adjustment @adzharamrullah

Comment (adzharamrullah) [reply components/AddHeroBanner/index.vue]: any suggestion ?

File: components/AddHeroBanner/index.vue (hunk 1 above)
Comment (adzharamrullah) [reply components/AddHeroBanner/index.vue:28]: hi @Ibwedagama, the syntax equal with `preventDefault()` an event which is event drag

Comment (Ibwedagama) [reply components/AddHeroBanner/index.vue:28]: Oh I see, it's just prevent default browser behavior but not triggering any function or method, thank you @adzharamrullah

File: components/AddHeroBanner/index.vue (hunk 2 above)
Comment (Ibwedagama) [reply components/AddHeroBanner/index.vue]: I think `Maaf file yang anda masukkan tidak didukung` or something like that? 😅 @adzharamrullah

Comment (adzharamrullah) [reply components/AddHeroBanner/index.vue]: i have done fixed it in [here](https://github.com/jabardigitalservice/dedi-cms/pull/13/commits/f8ab0039fabaf6789f69bc85a2e660a42f4ac2e8)

**Review by maulanayuseph — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

**Pull request merged**